metis-indexing/src/main/java/eu/europeana/indexing/fullbean/QualityAnnotationFieldInput.java

```diff
@@ -18,12 +18,9 @@
   public QualityAnnotationImpl apply(QualityAnnotation qualityAnnotation) {
     QualityAnnotationImpl qualityAnnotationImpl = new QualityAnnotationImpl();
     qualityAnnotationImpl.setAbout(qualityAnnotation.getAbout());
-    qualityAnnotationImpl.setDcTermsCreated(
-        FieldInputUtils.resourceOrLiteralListToArray(qualityAnnotation.getCreatedList()));
-    qualityAnnotationImpl.setOaHasTarget(
-        FieldInputUtils.resourceListToArray(qualityAnnotation.getHasTargetList()));
-    qualityAnnotationImpl.setOaHasBody(
-        FieldInputUtils.resourceListToArray(qualityAnnotation.getHasBodyList()));
+    qualityAnnotationImpl.setDcTermsCreated(qualityAnnotation.getCreated().getString());
+    qualityAnnotationImpl.setOaHasTarget(qualityAnnotation.getHasTarget().getResource());
+    qualityAnnotationImpl.setOaHasBody(qualityAnnotation.getHasBody().getResource());
```

Comment:
Please check for null in these three lines.

Reply:
Inderdaad!